vcnet/regex/engine.c

```diff
@@ -538,19 +538,16 @@ sopno lev;			/* PLUS nesting level */
 		while (m->g->strip[ss] != SOP(O_BACK, i))
```

Comment:
Short of explicit bug fixes, the regex code is something we've imported exclusively for Windows builds and is not code we actively maintain, so I'm not comfortable accepting changes to it lumped with everything else.